### DISCUSSION TITLE
Multi Audacity Versions

### DISCUSSION THREAD

**Discussion by TEOL2023** [intel/openvino-plugins-ai-audacity#285] [General]
I'd like OpenVinu to target from now on multiple versions of Audacity. I don't think it's fair that we have to redownload the OpenVinu or wait to get onto a major bug fix for recordings length without a crash just so my hissy mic jack can get some OpenVinu AI treatment.

I think OpenVinu updates needs to target multiple versions like next versions up but also if there is an update for updated plugins we can change to that version later on when it is out.

Systems like mine with hissy mic jacks rely on this but we also rely on these long recording bug fixes of new versions as well.

I'm still hoping for the noise reduction to do something about crackles of vinyl and hiss of cassette tapes without it removing the music.

For voice at the moment what it does is remove all the music other than voices which works to remove mic jack hiss from your own mic and that works great but some letters like the X in Mix and some S's sound mute which means even that needs some work as well.

I do hope that future versions would have us covered for .2 .3 ect. bug fix releases though and only focus on new versions when new plugin refinements are done.

It's so annoying that I really want this .2 fix for long recordings but have to stay on bugged .1 due to Vinu being out of date when maybe it shouldn't be detecting as out of date to begin with until a 3.7 series exists as maybe it should be targeting all 3.6 series installments like any changes of an .1 .2 .3 by making it target all possibilities of number revisions going by a normal Audacity setup.

3.5.0, 3.5.1, 3.5.2 seems to be the codes for 3.5 and 3.6 seems to be going by the same targets codes.

So what OpenVinu needs to do when the 3.7.0 series takes off whilst programming say that the Audacity version is 3.7.0, through to 3.7.9 that way we only need to get actual updates that updates the models as the version will target the possibilities spectrum in full for a 3.7 Vinu series to work on all of 3.7 after 2 days I really am not a fan of how Vinu does it now it's just not working like 3.6.1 the app I'm forced to say on until it's out the app doesn't really work for recordings on it.

**Comment by RyanMetcalfeInt8** [comment]
For the record, this is not something that these plugins really have control over. This needs to be a combined effort with the Audacity project to resolve. Right now, the Audacity app detects which version of the plugins we are compiled against and cross-checks it against the running version, and will simply deny them getting loaded if there is a mismatch in plugin version.
